console/console-init/ui/src/components/JsonEditor/JsonEditor.tsx

```diff
@@ -34,7 +36,11 @@ const JsonEditor: React.FunctionComponent<IJsonEditorProps> = ({
   const [value, setValue] = useState<string | undefined>(jsonValue);
 
   const onChange = (value: string) => {
-    setValue(value);
+    if (manageState) {
```

Comment:
please make changes as we had dicussed. use useEffect and don't need manageState